<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ShowImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>16</width>
      <height>501</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Vanish_Point">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>X axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="infobox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome!</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="int_scale">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vrml">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VRML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cal_bot">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Bottom</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cal_top">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Top</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cal_face">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Face</string>
    </property>
   </widget>
   <widget class="QPushButton" name="texture2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate 
Texture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Load_Image"/>
    <addaction name="separator"/>
    <addaction name="savecfg"/>
    <addaction name="loadcfg"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Load_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="savecfg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="loadcfg">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Load_Image</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onLoadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Vanish_Point</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_vanish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vrml</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_vrml()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cal_bot</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_bot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cal_top</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_top()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cal_face</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_face()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>texture2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_texture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savecfg</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadcfg</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLoadImage()</slot>
  <slot>on_vanish()</slot>
  <slot>on_texture()</slot>
  <slot>on_vrml()</slot>
  <slot>on_bot()</slot>
  <slot>on_top()</slot>
  <slot>on_face()</slot>
  <slot>save_config()</slot>
  <slot>load_config()</slot>
 </slots>
</ui>
